### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by robwise**
[Review on Reviewable](https://reviewable.io/reviews/shakacode/style-guide-javascript/4)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 
 1. ES7 static syntax for the propTypes and defaultProps
-2. Use lodash `_.bindAll` for callback bindings.
+1. Use lodash `_.bindAll` for callback bindings.
 
-For example:
+An example React component:
 
-```javascript
+```js
```
Comment (robwise) [inline README.md:25]: sorry for noise, was trying something and thought I changed it back, either way works tho

**Comment by alex35mil** [comment]
:lgtm: 

---

Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

_[README.md, line 10 [r2]](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/4#-K9915uIfAbch-axQX3W:-K9915uJRn3sXJKTg3P1:-713877879) ([raw file](https://github.com/shakacode/style-guide-javascript/blob/c19f62748ee6e00b1af9c56199e4dfb44796cb85/README.md#L10)):_
It's already 100.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/4#-:-K991VZEDWLcKv6YJFaP:980077423)_

**Comment by robwise** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

_[README.md, line 10 [r2]](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/4#-K9915uIfAbch-axQX3W:-K9EEyCqxLpIdyAlfceg:1613698602) ([raw file](https://github.com/shakacode/style-guide-javascript/blob/c19f62748ee6e00b1af9c56199e4dfb44796cb85/README.md#L10)):_
@alexfedoseev what's already 100? Please let me know what I need to change

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/4)_

**Comment by justin808** [comment]
:lgtm_strong:  But change the 120 to 100.

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

_[README.md, line 10 [r2]](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/4#-K9915uIfAbch-axQX3W:-K9EnqZ1HOdzStQxDhKG:1049361542) ([raw file](https://github.com/shakacode/style-guide-javascript/blob/c19f62748ee6e00b1af9c56199e4dfb44796cb85/README.md#L10)):_
It comes as 100 from airbnb

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/4#-:-K9Eo4sQwdujf0nrjUR8:1728994366)_

**Comment by robwise** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

_[README.md, line 10 [r2]](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/4#-K9915uIfAbch-axQX3W:-K9G2q8jXguVci9Mckbv:-499146009) ([raw file](https://github.com/shakacode/style-guide-javascript/blob/c19f62748ee6e00b1af9c56199e4dfb44796cb85/README.md#L10)):_
Done.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/4)_

**Comment by justin808** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/style-guide-javascript/4)_

**Pull request merged**